R/get_14cpalaeolithic.R

```diff
@@ -0,0 +1,60 @@
+#' @rdname db_getter_backend
+#' @export
+get_14cpalaeolithic <- function(db_url = get_db_url("14cpalaeolithic")) {
+
+  check_connection_to_url(db_url)
+
+  # db_url <-"https://ees.kuleuven.be/geography/projects/14c-palaeolithic/radiocarbon-palaeolithic-europe-database-v26-extract.xlsx"
+
+
+  # download data
+  temp <- tempfile(fileext=".xlsx")
+
+  utils::download.file(url = db_url, destfile = temp, mode="wb")
+
+  # Read each file and write it to csv
+  lapply(temp, function(f) {
+    df = openxlsx::read.xlsx(f, sheet=1)
+    utils::write.csv(df, gsub("xlsx", "csv", f), row.names=FALSE)
+  })
+
+  db_path_xlsx <- unlist(strsplit(temp, split = ".", fixed = TRUE))
+  db_path_csv <- paste(db_path_xlsx[1], ".csv", sep = "")
+
+  # read data
+  db_raw <- data.table::fread(
+    db_path_csv,
+    sep = ",",
+    encoding = "UTF-8",
+    colClasses = "character",
```

Comment:
So this reads every column as character? That could probably do as well into other parser functions, or? Less verbose.